### DISCUSSION TITLE
com.canonical.AppMenu.Registrar - safe or not?

### DISCUSSION THREAD

**Discussion by lucasmz-dev** [netblue30/firejail#6570] [Q&A]
Same question as #6569, same case

(Is there a way to actually just audit these myself easily at a basic level so I don't have to ask around?)

**Comment by rusty-snake** [comment]
> (Is there a way to actually just audit these myself easily at a basic level so I don't have to ask around?)

busctl, d-spy, ...

> com.canonical.AppMenu.Registrar - safe or not?

Unless you actually use it, I would just revoke access to it.
